Lib/Emscripten/Emscripten.cpp

```diff
@@ -623,6 +624,10 @@ DEFINE_INTRINSIC_FUNCTION(asm2wasm, "i32s-div", I32, I32_divs, I32 left, I32 rig
 {
 	return left / right;
 }
+DEFINE_INTRINSIC_FUNCTION(asm2wasm, "f64-rem", F64, F64_rems, F64 left, F64 right)
+{
+	return (F64)remainder(left, right);
```

Comment:
I think this should call `fmod` instead of `remainder`: Emscripten implements it with the JavaScript `%` operator, and according to [the JavaScript spec](https://www.ecma-international.org/ecma-262/5.1/#sec-11.5.3), the behavior of that operator on floating point values is like the C `fmod` function.